recipes/ipserver/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "ipserver" %}
+{% set version = "1.0.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Technically conda-forge only supports `python>=3.9` via the global `python_min`. I'm going to let this remain at 3.8, but just be aware that it could cause issues down the road.